presto-native-execution/presto_cpp/main/PrestoServer.cpp

```diff
@@ -99,6 +101,13 @@ std::string stringifyConnectorConfig(
   return out.str();
 }
 
+bool isCacheTtlEnabled() {
+  const auto* systemConfig = SystemConfig::instance();
+  return systemConfig->cacheVeloxTtlEnabled() &&
+      systemConfig->cacheVeloxTtlThreshold() > seconds::zero() &&
+      systemConfig->cacheVeloxTtlCheckInterval() > seconds::zero();
```

Comment:
Should we solely rely on cacheVeloxTtlEnabled flag for this method and add sanity check when setting other configs so that it throws if ttl is enabled but other settings are 0? This can cause silent failure and introduce troubles when debugging.

Reply:
Revised